components/alertbanner/index.css

```diff
@@ -0,0 +1,128 @@
+  --spectrum-alert-banner-text-to-button-horizontal: var(--spectrum-spacing-300);
+  --spectrum-alert-banner-text-to-divider: var(--spectrum-spacing-300);
+  --spectrum-alert-banner-top-icon: var(--spectrum-alert-banner-top-to-workflow-icon);
+  --spectrum-alert-banner-top-text: var(--spectrum-alert-banner-top-to-text);
+  --spectrum-alert-banner-bottom-text: var(--spectrum-alert-banner-bottom-to-text);
+  --spectrum-alert-banner-close-button-spacing: var(--spectrum-spacing-100);
+  --spectrum-alert-banner-edge-to-divider: var(--spectrum-spacing-100);
+  --spectrum-alert-banner-edge-to-button: var(--spectrum-spacing-100);
+  --spectrum-alert-banner-text-to-button-vertical: var(--spectrum-spacing-100);
+
+  /* colors */
+  --spectrum-alert-banner-informative-background: var(--spectrum-informative-background-color-default);
+  --spectrum-alert-banner-negative-background: var(--spectrum-negative-background-color-default);
+  --spectrum-alert-banner-font-color: var(--spectrum-white);
+
+  .spectrum--large & {
+    --spectrum-alert-banner-close-button-spacing: var(--spectrum-spacing-200);
+    --spectrum-alert-banner-edge-to-divider: var(--spectrum-spacing-200);
+    --spectrum-alert-banner-edge-to-button: var(--spectrum-spacing-200);
+    --spectrum-alert-banner-text-to-button-vertical: var(--spectrum-spacing-200);
+  }
+}
+
+.spectrum-AlertBanner {
+  display: none;
+
+  &.is-open {
+    display: flex;
+  }
+
+  justify-content: space-between;
+  inline-size: var(--mod-alert-banner-size, var(--spectrum-alert-banner-size));
+  min-block-size: var(--mod-alert-banner-min-height, var(--spectrum-alert-banner-min-height));
+
+  font-size: var(--mod-alert-banner-font-size, var(--spectrum-alert-banner-font-size));
+  color: var(--mod-alert-banner-font-color, var(--spectrum-alert-banner-font-color));
+
+  background-color: var(--mod-alert-banner-netural-background, var(--spectrum-alert-banner-netural-background));
+
+  &--info {
+    background-color: var(--mod-alert-banner-informative-background, var(--spectrum-alert-banner-informative-background));
+  }
+
+  &--negative {
+    background-color: var(--mod-alert-banner-negative-background, var(--spectrum-alert-banner-negative-background));
+  }
+
+  .spectrum-Divider--vertical {
+    margin-block: var(--mod-alert-banner-edge-to-divider, var(--spectrum-alert-banner-edge-to-divider));
+    height: auto;
+    align-self: stretch;
+  }
+
+  .spectrum-Button {
+    margin-block: var(--mod-alert-banner-edge-to-button, var(--spectrum-alert-banner-edge-to-button));
+    margin-inline-end: var(--mod-alert-banner-text-to-divider, var(--spectrum-alert-banner-text-to-divider));
+    margin-inline-start: auto;
+  }
+
+  .spectrum-CloseButton {
+    margin-inline: var(--mod-alert-banner-close-button-spacing, var(--spectrum-alert-banner-close-button-spacing));
+    margin-block-start: var(--mod-alert-banner-close-button-spacing, var(--spectrum-alert-banner-close-button-spacing));
+    align-self: flex-start;
+  }
```

Comment:
I think we may want to consider using the custom properties approach that we discussed for the DropZone component instead of using these child selectors.